schemas/health/en/cris_0001.json

```diff
@@ -574,7 +574,7 @@
       "title": "About you",
       "summary": {
         "show_on_completion": false,
-        "page_title": "New section - Introduction",
+        "page_title": "New section- Introduction",
```

Comment:
This looks like a mistake. Can you check with CRIS author or change it back. The others aren't like this. I want to avoid another deploy to fix this.

Reply:
No that is a mistake, it looks like the author removed it in this new survey but it was on the old one with the space.

Cheers, Mebin I missed that one.